src/Input/components/InnerInput.js

```diff
@@ -0,0 +1,44 @@
+import atomize from '@quarkly/atomize';
+
+export default atomize.input(
+    {
+        name: 'InnerInput',
+        description: {
+            en: 'Input element',
+            ru: 'Input element',
+        },
+        effects: {
+            hover: ':hover',
+            focus: ':focus',
+            required: ':required',
+            placeholder: '::placeholder',
+            invalid: ':invalid',
+            disabled: ':disabled',
+            firstChild: ':first-child',
+            lastChild: ':last-child',
+            onlyChild: ':only-child',
+            firstOfType: ':first-of-type',
+            lastOfType: ':last-of-type',
+            onlyOfType: ':only-of-type',
+        },
+        propInfo: {},
+        useAliases: false,
```

Comment:
Potentially, such a solution could bring problems to nested elements, but it's okay for now.